app/worker.py

```diff
@@ -70,11 +70,14 @@ def handler(event, context):
             current_components[cid]['status'] = data['status']
             changed.append(current_components[cid]['name'])
 
-    # If components stop reporting set their status to warning.
+    # If components stop reporting set their status to warning if not already
+    # set.
     for cid in current_components.keys():
```

Comment:
NIT: just noticed this unnecessary use of the `keys` method from a previous commit